internal/graph/graph.go

```diff
@@ -442,13 +447,24 @@ func NewLayeredCheckResolver(
 
 	localCheckResolver.SetDelegate(cycleDetectionCheckResolver)
 
+	var dispatchThrottlingCheckResolver *DispatchThrottlingCheckResolver
+	if throttlingEnabled {
```

Comment:
Please see logic of https://github.com/openfga/openfga/blob/main/pkg/server/server.go#L369-L405 (order of dispatch resolver )